<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyStatusDialogClass</class>
 <widget class="QDialog" name="ModifyStatusDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Status</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>cmbShow</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>lneName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblPriority">
       <property name="text">
        <string>Priority</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>spbPriority</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cmbShow"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lneName"/>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spbPriority">
       <property name="minimum">
        <number>-127</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pteText"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cmbShow</tabstop>
  <tabstop>lneName</tabstop>
  <tabstop>spbPriority</tabstop>
  <tabstop>pteText</tabstop>
  <tabstop>dbbButtons</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
